Task 1/RAINBOWA/Question.md

```diff
@@ -0,0 +1,58 @@
+Chef likes all arrays equally. But he likes some arrays more equally than others. In particular, he loves Rainbow Arrays.
+
+An array is Rainbow if it has the following structure:
+
+First a1 elements equal 1.
+Next a2 elements equal 2.
+Next a3 elements equal 3.
+Next a4 elements equal 4.
+Next a5 elements equal 5.
+Next a6 elements equal 6.
+Next a7 elements equal 7.
+Next a6 elements equal 6.
+Next a5 elements equal 5.
+Next a4 elements equal 4.
+Next a3 elements equal 3.
+Next a2 elements equal 2.
+Next a1 elements equal 1.
+ai can be any non-zero positive integer.
+There are no other elements in array.
+
+Help Chef in finding out if the given array is a Rainbow Array or not.
+
+Input
```

Comment:
Here also 👍

Reply:
Oh I just got to know about this I will do these changes thankyou